### PR TITLE
Glossary

### LINKED ISSUE #309: Create a Glossary
There may be developers reading our docs who are not familiar with some terms that we use, such as _opinionated framework, continuous delivery pipeline,_ or _reactive systems._

We should create a glossary that would explain such terms based on the expected knowledge level of the target audience. 

This glossary should be shared across the whole doc set.

*Possible sub-task: Get better description of the target audience from the stakeholders.*

### PR DESCRIPTION
**Pull request by tradej**
A short glossary with basic term explanations, all in one place. The explanations are the best I could come up with given my limited knowledge of the product, so please feel free to go to town on them.

The intention is to include the glossary at the end of all books for quick reference.

I have decided against implementing a glossary of terms unrelated to the product just yet, and try to reword the sections those terms if possible instead (see #670). We can discuss this with Ali in regard to long-term content strategy.

Resolves #309.

### CODE REVIEW

File: docs/topics/appendix-glossary.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+== Terms Specific to {launcher}
+
+[glossary_booster]
+Booster:: An language-specific implementation of a particular xref:glossary_mission[mission] on a particular xref:glossary_runtime[runtime]. Boosters are listed in a xref:glossary_booster_catalog[booster catalog].
++
+For example, a booster is a web service with a REST API implemented using the {WildFlySwarm} runtime.
+
+[[glossary_booster_catalog]]
+Booster Catalog:: An Git repository that contains information about boosters.
```
Comment (Ladicek) [inline docs/topics/appendix-glossary.adoc]: Is that "a Git repository" instead of "an"?

**Comment by Ladicek** [comment]
Otherwise LGTM.

File: docs/topics/appendix-glossary.adoc (hunk 1 above)
Comment (tradej) [reply docs/topics/appendix-glossary.adoc]: Ha, good catch, thanks.

**Comment by rhoads-zach** [comment]
merged: 38beeb6541948a49a2ea52a93b291d5a087563ca

**Pull request closed without merging**